doc/Requirements Specification/Parts/Functional Requirements.lyx

```diff
@@ -685,8 +685,9 @@ name "/OF130/"
 \end_inset
 
  The user may pause and resume a measurement.
- Pausing causes all measurement activity to stop.
- Resuming continues the measurement from where they were paused.
+ Pausing causes all analysis activity to stop.
+ Resuming continues the analysis from the beginning of the measurement it
+ was taking when it had been paused.
```

Comment:
Are you sure, that no comma has to be in this sentence? (I'm not, but I might be wrong)